src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views_component.tsx

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
nit: the standard we've adopted here so far has been: `no_data_views.component.tsx`